src/m365/spo/commands/serviceprincipal/serviceprincipal-permissionrequest-list.ts

```diff
@@ -58,6 +60,32 @@ class SpoServicePrincipalPermissionRequestListCommand extends SpoCommand {
       this.handleRejectedPromise(err);
     }
   }
+
+  private async getoAuth2Permissiongrants(sPOClientExtensibilityWebApplicationPrincipalId: string): Promise<string[]> {
+    const requestOptions: CliRequestOptions = {
+      url: `https://graph.microsoft.com/beta/oAuth2Permissiongrants/?$filter=clientId eq '${sPOClientExtensibilityWebApplicationPrincipalId}' and consentType eq 'AllPrincipals'`,
```

Comment:
Since we're using a beta endpoint, let's add a remark to command's docs about it like we do for other commands that use beta endpoints.